rasa_core/policies/embedding_policy.py

```diff
@@ -269,13 +269,13 @@ def _load_visual_params(self, config: Dict[Text, Any]) -> None:
         self.evaluate_every_num_epochs = config['evaluate_every_num_epochs']
         if self.evaluate_every_num_epochs < 1:
             self.evaluate_every_num_epochs = self.epochs
-
         self.evaluate_on_num_examples = config['evaluate_on_num_examples']
 
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
-        config = copy.deepcopy(self.defaults)
+        config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```

Comment:
we don't need `self.tf_defaults` anymore, because if there is no `tf_config`, we init `tf.Session` with `None`